deploy_dod_dataset.py

```diff
@@ -1,10 +1,17 @@
 from io import BytesIO
 import boto3
 import os
+import logging
 
+from libs.build_params import OUTPUT_DIR_DOD
+from libs.constants import INTERVENTIONS
+from libs.validate_results import validate_states_df, validate_counties_df, validate_states_shapefile, validate_counties_shapefile
```

Comment:
this is a nit fersure, but I think it's better to make this `from libs import validate_results` - that way in the code, it's clear where calling those functions comes from.
https://google.github.io/styleguide/pyguide.html#22-imports has a good piece about that.